distributed/scheduler.py

```diff
@@ -851,6 +852,7 @@ def __init__(
         dashboard_address=None,
         preload=None,
         preload_argv=(),
+        resources_auto={},
```

Comment:
I think we want this to be `None` so we'll pull from `dask.config`

```suggestion
        resources_auto=None,
```